docs/guide.md

```diff
@@ -0,0 +1,104 @@
+# Install openebs-addon-only
+
+### Prerequisite
+
+- Already installed prometheus-operator stack
+- OpenEBS installed
+
+---
+**NOTE**
+
+To allow Prometheus to discover all `PodMonitors/ServiceMonitors`, without applying label filtering, you have to set `prometheus.prometheusSpec.podMonitorSelectorNilUsesHelmValues` and `prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues` to `false`
+
+---
+
+## Step by Step Process
+
+1. **Update values.yaml**
+   
+   -  Disable dependent chart(kube-prometheus-stack) by updating `kube-prometheus-stack.install` to `false`
+   -  Set `openebsMonitoringAddon.enabled` to `true` 
+   -  If you have updated the label for grafana sidecar container(`grafana-sc-dashboard`), then update `openebsMonitoringAddon.grafana.sidecar.dashboards.label: <UPDATED_LABEL>`
+   -  Update `namespaceOverride` with the namespace name in which prometheus stack is installed
+   -  After updating these specific values in values.yaml, these fields should look like this(example) :
+		```console
+		...
+		namespaceOverride: "prometheus-operator"
+		openebsMonitoringAddon:
+		  enabled: true
+		  grafana:
+		    ## Deploy custom openebs dashboards
+		    customDashboardsEnabled: true
+		    ## Sidecar container to load dashboard cm
+		    sidecar:
+			  dashboards:
+			  enabled: true
+			  # ConfigMaps with label below will be added to Grafana as dashboards.
+			  label: grafana_dashboard
+
+		kube-prometheus-stack:
+		  install: false
+		...
+		```
+
+2. **Install grafana plugin**
```

Comment:
Probably these grafana customizations should be in pre-requisites.

Reply:
Done.